tests/next_tests/integration_tests/feature_tests/ffront_tests/test_import_from_mod.py

```diff
@@ -0,0 +1,36 @@
+# GT4Py - GridTools Framework
+#
+# Copyright (c) 2014-2024, ETH Zurich
+# All rights reserved.
+#
+# Please, refer to the LICENSE file in the root directory.
+# SPDX-License-Identifier: BSD-3-Clause
+
+from tests.next_tests.integration_tests.feature_tests.ffront_tests import dimensions_mod as dims
+from tests.next_tests.integration_tests.cases import IDim, KDim
```

Comment:
```suggestion
```
then see comment at `domain=...`

Reply:
Something is still wrong in the implementation (didn't investigate yet), because if I remove the second line `from tests.next_tests.integration_tests.cases import IDim, KDim` which is not used in this file, the test breaks.